flowauth/backend/tests/test_commands.py

```diff
@@ -64,16 +64,73 @@ def test_init_db_force(app):
             add_admin_command, ["DUMMY_ADMINISTRATOR", "DUMMY_ADMINISTATOR_PASSWORD"]
         )
         assert len(User.query.all()) > 0
+        app.config["DB_IS_SET_UP"].clear()
         result = runner.invoke(init_db_command, ["--force"])
         assert len(User.query.all()) == 0
 
 
+def test_demo_data_only_sets_up_once(app, caplog):
+    """
+    Demo data should only be created once.
+    """
+    with app.app_context():
+        runner = app.test_cli_runner()
+        app.config["DB_IS_SET_UP"].clear()
+        result = runner.invoke(demodata)
```

Comment:
Just to double-check: these two instances of `runner.invoke(demodata)` will run in serial, i.e. the first one will definitely finish before the second one is run? Just want to make sure this test can't accidentally fail if they run concurrently.

Reply:
Definitely serial.